objects/rct2/ride/rct2.ding1.json

```diff
@@ -65,7 +65,7 @@
             "ru-RU": "Шлюпки"
         },
         "description": {
-            "en-GB": "Riders travel in inflatable dinghies down a twisting semi-circular or completely enclosed tube track",
+            "en-GB": "Inflatable dinghies that are twisting down a semi-circular or completely enclosed tube track",
```

Comment:
This still seems to be written from the track's perspective.

Reply:
It should be "Inflatable dinghies that can twist down a semi-circular or completely enclosed tube track"